charts/project-origin-wallet/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: project-origin-wallet
 description: A helm chart for deploying the Project Origin Wallet
 type: application
-version: 0.1.0
-appVersion: "0.1.0"
+version: 0.1.1
+appVersion: "0.1.1"
```

Comment:
There is no reason to change this, this is changed as part of the build pipeline, but a value is required for test to pass.

Reply:
Done:

https://github.com/project-origin/wallet/pull/254/commits/0f64ae7c5fe5c2dc08defa73733c130ad777c7c5